apps/docs/packages/telegram-apps-sdk-vue/2-x.md

```diff
@@ -0,0 +1,146 @@
+---
```

Comment:
Please, rename this file to `1-x.md`. This will be the first major release of this package.

I got the possible idea like "sdk major version = sdk-vue major version", but we have a separate versioning here.

Also, a new entry in the `.vitepress` file should be added, so the documentation will be accessible in the side menu

![image](https://github.com/user-attachments/assets/392fe193-372b-4ed7-a59f-ea43af8e093c)

Reply:
Also, I removed directory vue-sdk. Did it similar with router-integration pages.